containers/Investment/RevolvingPool/View/CollectCard.tsx

```diff
@@ -1,56 +1,86 @@
 import * as React from 'react'
 import { Box, Button, Heading, Table, TableBody, TableRow, TableCell } from 'grommet'
 import { Pool } from '../../../../config'
+import { toPrecision } from '../../../../utils/toPrecision'
+import { addThousandsSeparators } from '../../../../utils/addThousandsSeparators'
+import { baseToDisplay } from '@centrifuge/tinlake-js'
+import { createTransaction, TransactionProps } from '../../../../ducks/transactions'
+import { connect } from 'react-redux'
 
 import { Description } from './styles'
 import { Card } from './TrancheOverview'
 
-interface Props {
+interface Props extends TransactionProps {
   pool: Pool
   tranche: 'senior' | 'junior'
   setCard: (card: Card) => void
+  disbursements: any
+  tinlake: any
 }
 
 const CollectCard: React.FC<Props> = (props: Props) => {
   const token = props.tranche === 'senior' ? 'DROP' : 'TIN'
 
+  const type = props.disbursements.payoutCurrencyAmount.isZero() ? 'Redeem' : 'Invest'
```

Comment:
Shouldn't this be the other way around? Redeem if the currrency amount is not zero?

Reply:
You're right, nice find!